packages/shared/src/lib/core/profile/migrations/alpha/alpha-profile-migration-map.ts

```diff
@@ -48,5 +49,6 @@ export const ALPHA_PROFILE_MIGRATION_MAP: ProfileMigrationMap = {
     20: alphaProfileMigration20To21,
     // ^^^ release 1.0.4 ^^^
     21: alphaProfileMigration21To22,
+    22: alphaProfileMigration22To23,
     // ^^^ release 1.0.8 ^^^
```

Comment:
this migration is part of the next release
```suggestion
    // ^^^ release 1.0.8 ^^^
    22: alphaProfileMigration22To23,
```